pages/design/atomic-template-empty.md

```diff
@@ -28,7 +28,10 @@ accessibility_requirements: # Remove whatever is not applicable
 input_aspects:
   -  # (e.g. HTTP Messages, DOM Tree, CSS Styling, Accessibility Tree, Language, etc.,)
 
-authors:
+acknowledgements:
+  authors:
+  -  # (full names as found in `contributors` property in  `package.json` - if not yet listed, please have authors added to the list)
+  previous_authors:
```

Comment:
We can potentially have `reviewers`, `funding` etc., It may be nice to add notes on what else can be in acknowledgements.

Reply:
> We can potentially have `reviewers`, `funding` etc., It may be nice to add notes on what else can be in acknowledgements.

As far as I know, we have not yet agreed on what exactly should be there in addition to authors and previous authors, so I think adding such notes at this time would be a little confusing.

I guess consensus is that a previous author is somebody who is no longer actively writing the rule. However, I would find it difficult to decide what to do with reviewers. A reviewer is somebody who has reviewed a comma, and somebody who is actively cooperating with the rule writers. Should we acknowledge both at the same level? I guess we need to talk about this and then adapt the notes accordingly.

Reply:
This conversation seems to be a duplicate. I'm resolving it for the sake of clarity.